javafx/resources/css/nocfa.css

```diff
@@ -21,6 +21,36 @@
 	-fx-background-image: url(nocfa/stop_hover.png);
 }
 
+/* Playlist controls */
+
+#shuffle:selected, #shuffleon {
+	-fx-background-image : url(nocfa/shuffle.png);
+}
+
+#shuffle, #shuffleoff {
+	-fx-background-image : url(nocfa/shuffle-disabled.png);
+}
+
+#shuffle:selected:hover, #shuffleon:hover {
+	-fx-background-image : url(nocfa/shuffle-hover.png);
+}
+
+#shuffle:hover, #shuffleoff:hover {
+	-fx-background-image : url(nocfa/shuffle-disabled-hover.png);
+}
+
+#repeat:selected, #repeaton {
+	-fx-background-image: url(nocfa/refresh.png);
+}
+
+#repeat:selected:hover, #repeaton:hover {
+	-fx-background-image: url(nocfa/refresh_hover.png);
+}
+
+#repeat, #repeatoff {
```

Comment:
what is #repeatoff and #repeaton? You do not change the CSS ID to adjust for state change - that's what states are for.